src/documentation/doc-util/doc-query.ts

```diff
@@ -12,17 +12,24 @@ import { printAsMs } from '../../util/text/time';
 import { asciiSummaryOfQueryResult } from '../../queries/query-print';
 import { FlowrAnalyzerBuilder } from '../../project/flowr-analyzer-builder';
 import { getReplCommand } from './doc-cli-option';
+import type { SingleSlicingCriterion } from '../../slicing/criterion/parse';
 
 export interface ShowQueryOptions {
-	readonly showCode?:       boolean;
-	readonly collapseResult?: boolean;
-	readonly collapseQuery?:  boolean;
+	readonly showCode?:         boolean;
+	readonly collapseResult?:   boolean;
+	readonly collapseQuery?:    boolean;
+	readonly shorthand?:        string;
+	readonly multipleCriteria?: boolean;
```

Comment:
Having a `multipleCriteria` alongside `shorthand` feels not good, why not move this to `sliceQueryShorthand`?

Reply:
I expanded the examples to include multiple criteria.
Let me know if you had something else in mind 🙂

Reply:
I would have moved the explanation string to be generated by the sliceShorthand :D, besides this this PR is fine, shall I take care of the conflicts?

Reply:
I fixed them :)

Reply:
Great thank you :D - so.. merge on Sunday? :D